packages/nouns-webapp/src/utils/getNounsVotes.ts

```diff
@@ -1,14 +1,15 @@
+interface NounVote {
```

Comment:
nit:

Similar comment to the one I made on the `DelegateNouns` interface ... this isn't _really_ a Noun vote right? It's the votes cast by a delegate with the represented Nouns ... right? Again, not blocking but wonder if naming could make this clearer.